src/index.html

```diff
@@ -7,7 +7,6 @@
 
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
-  <script src="node_modules/openlayers/dist/ol-debug.js"></script>
   <link rel="stylesheet" href="node_modules/openlayers/dist/ol-debug.css" type="text/css">
```

Comment:
we need to move it sometime (need research how to do it ) I think we can tell webpack in some way how to handle this take a look how I did it on event drops